sympy/polys/domains/tests/test_polynomialring.py

```diff
@@ -23,20 +23,16 @@ def test_globalring():
     assert 1/x not in R
     assert 1/(1 + x) not in R
     assert Y in R
-    assert X.ring == R
     assert X * (Y**2 + 1) == R.convert(x * (y**2 + 1))
-    assert X * y == X * Y == R.convert(x * y) == x * Y
-    assert X + y == X + Y == R.convert(x + y) == x + Y
-    assert X - y == X - Y == R.convert(x - y) == x - Y
     assert X + 1 == R.convert(x + 1)
     raises(ExactQuotientFailed, lambda: X/Y)
-    raises(ExactQuotientFailed, lambda: x/Y)
-    raises(ExactQuotientFailed, lambda: X/y)
+    raises(TypeError, lambda: x/Y)
+    raises(TypeError, lambda: X/y)
```

Comment:
Since `DMP` no longer has `.ring` it cannot interact directly with Expr like this (it does not know what its generators are).